error.go

```diff
@@ -0,0 +1,35 @@
+package main
```

Comment:
That seems clearer, though I'll use the more idiomatic `fetch_error` name (with underscore).